cluster/deployment/config.yaml

```diff
@@ -50,6 +50,8 @@ monitoring:
       loadTester:
         minRate: 0.95
     logAlerts:
+      # TODO(hyperledger-labs/splice#1483): stop ignoring sequencer connection warnings once they have better signal
```

Comment:
I didn't pay attention to the PRs but why is this not in the internal repo?

Reply:
tech debt #1490